### ISSUE TITLE
[error] Some states could not be loaded as their base is not available, check for typos or cycles

### ISSUE THREAD

**Issue by robotlinker** [rohanpsingh/grasp-fsm-sample-controller#3]
manual added GraspFSM.yaml file under /usr/lib/x86_64-linux-gnu/mc_controller/GraspFSM/states/data$  directory and launch mc_mujoco shows the following errors:

![Screenshot from 2023-03-31 11-38-15](https://user-images.githubusercontent.com/24693929/229016850-7014b0c9-350d-49e5-8442-53b1288caa0c.png)

**Comment by rohanpsingh** [comment]
Did you modify `GraspFSM.in.yaml` in any way? Why are you adding the file manually?

**Comment by robotlinker** [comment]
If I don't manual add the file, it will show following errors:
 

![Screenshot from 2023-03-31 11-33-48](https://user-images.githubusercontent.com/24693929/229396198-fdec40bd-29c3-4231-ad38-652b53b55c22.png)

**Comment by robotlinker** [comment]
Looks like after make install the package, it didn't install files under : /usr/lib/x86_64-linux-gnu/mc_controller/GraspFSM/states/ 
  which shows states data does not exist
![Screenshot from 2023-04-03 10-24-53](https://user-images.githubusercontent.com/24693929/229397558-0fabcd92-4126-4efe-b475-a73d7ab4cba5.png)

**Comment by rohanpsingh** [comment]
Were you able to successfully build and install `LIPMWalking` (https://github.com/rohanpsingh/grasp-fsm-sample-controller/issues/2)?

**Comment by robotlinker** [comment]
Yes, I make and installed the LIPMWalking package, but a different branch as you have mentioned before.

![Screenshot from 2023-04-03 10-59-20](https://user-images.githubusercontent.com/24693929/229401641-54c81944-02fe-42d8-ad65-cf2f75b32716.png)

**Comment by rohanpsingh** [comment]
Thanks, could you also please the following things:
- Are you able to run any other `mc_rtc` controller? You can try `Posture` or `CoM`
- Are you able to run `LIPMWalking` controller?
- Do you have access to the `HRP5P` robot?
